.env.example

```
@@ -11,3 +11,11 @@ ROLLBAR_ENV=development
 
 # TODO: Replace `rails-template` with the name of the app.
 DATABASE_URL=postgres://postgres@localhost:5432/rails-template-development
+
+# TODO: Replace `example.com` with the canonical hostname of the app
+CANONICAL_HOSTNAME=example.com
+
+# TODO: Replace `example.org` with a comma seperated list of any
+# other hostnames you want to app to respond to and redirect to the canonical
+# hostname, or delete this line completely
+ADDITIONAL_HOSTNAMES=example.org
```

Can we suppport an "empty" value of this and leave it blank, so:
```suggestion
ADDITIONAL_HOSTNAMES=
```

That way you don't have to remember how to put it back in when you actually release.

---

Done! 👍